home/views.py

```diff
@@ -29,19 +31,27 @@ def details_view(request, id):
 
 
 def search_view(request, page: str = "1"):
-    query = request.GET.get("query", "")
+
+    new_search = request.GET.get("new", "")
```

Comment:
This is a quick fix for search queries persisting when they shouldn't.

It's bypassing behaviour caused by `request.session["query"] = query` on line 94

 I think we will be able to remove this again after some refactoring, but don't want to do this without tests!